### PR TITLE
Add changeLog section in PR template

### PR DESCRIPTION
**Pull request by daskyrk**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:
In order to extract data to automatically generate changeLog.

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
 
+
+#### ChangeLog
+<!--
+Describe the specific changes from the user's perspective, as well as possible Breaking Change and other risks.
+-->
+
+| Language | Changelog |
+| --------- | ------------ |
+| 🇺🇸 English |              |
```
Comment (sfwn) [inline .github/PULL_REQUEST_TEMPLATE.md:30]: I think it's duplicated with `#### What this PR does / why we need it:`

**Review by sfwn — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,22 +1,36 @@
...
 
-Add one of the following kinds:
-/kind feature
-/kind bug
-/kind polish
-/kind document
+- [ ] feature
+- [ ] bug
+- [ ] polish
+- [ ] document
```
Comment (Effet) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: I hate checkbox, pr's checkbox will become sub-tasks, they aren't sub-tasks!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Effet) [reply .github/PULL_REQUEST_TEMPLATE.md:30]: agree, pr's type already tell us how this pr related to changelog.

**Review by Effet — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (daskyrk) [reply .github/PULL_REQUEST_TEMPLATE.md:30]: `What this PR does` is required, but may not required in changeLog.
If duplicated, just copy to this section.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (daskyrk) [reply .github/PULL_REQUEST_TEMPLATE.md]: Have reverted

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Effet) [reply .github/PULL_REQUEST_TEMPLATE.md:31]: in this case, how you ensure that `Changelog` is really needed? (I am not talking about standard and mandatory requirement, I am talking about execution result)

**Review by Effet — APPROVED** [review]

**Review by Effet — DISMISSED** [review]
sorry miss `approved`

**Comment by Effet** [comment]
> #### What type of this PR
> Add one of the following kinds:
> /kind feature
> 
> #### What this PR does / why we need it:
> In order to extract data to automatically generate changeLog.
> 
> #### Which issue(s) this PR fixes:
> * Fixes #your-issue_number
> * [Erda Cloud Issue Link](paste your link here)
> 
> #### Specified Reviewers:
> /assign @your-reviewer
> 
> #### Need cherry-pick to release versions?
> Add comment like `/cherry-pick release/1.0` when this PR is merged.
> 
> > For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

like your pr not fulfill all of field, who can ensure every one fulfill `changelog`

**Pull request merged**